src/common/autosetup.js

```diff
@@ -142,41 +142,26 @@ function goToNextDay(loadDelay, loadMaxAttempts, action) {
 }
 
 function getRosterRows() {
-    const tableBody = document.querySelector('.players-table tbody.Table__TBODY');
+    const tableBody = document.querySelectorAll('.players-table tbody.Table__TBODY');
     const starterRows = [];
     const benchRows = [];
-    for (const row of tableBody.getElementsByTagName('tr')) {
-        const slotCell = row.getElementsByTagName('td')[0];
-        const slotType = parseSlotType(slotCell.textContent);
-        if (slotType === SLOT_TYPE_BENCH || slotType === SLOT_TYPE_IR) {
-            benchRows.push(row);
-        } else {
-            starterRows.push(row);
+    const playerTable1 = tableBody[0];
+    const playerTable2 = tableBody.length > 3 ? tableBody[3] : [];
+
+    [playerTable1, playerTable2].forEach((table) => {
+        for (const row of table.getElementsByTagName('tr')) {
```

Comment:
Ah whoops, I forgot that `tableBody`'s elements were `NodeList`s, not arrays! Gotta love cases where you can't access the page to test :laughing: .

I think I have fixed this in e9892a4f41e49fb00e3c95536189dc41588fc229.

Reply:
Yeah, it's an interesting way to write code. 😆

Testing this extension is a mess in general. Sometimes I have to simply wait for a certain combination of players and games to show up in order to test corner cases. Between seasons it usually breaks because the page layout changes.

It would probably be possible to set up some sort of test workflow with a mocked version of the HTML and server, but to be honest I don't want to put too much work into an extension that ESPN could so easily replace with a natively implemented version.

Anyway, the code seems to work just fine now, so I'll merge this!